adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -82,8 +85,10 @@ trait AkkaHttpAdapter {
 
   def makeWebSocketService[R, E](
     interpreter: GraphQLInterpreter[R, E],
-    skipValidation: Boolean = false
-  )(implicit ec: ExecutionContext, runtime: Runtime[R], materializer: Materializer): Route = {
+    skipValidation: Boolean = false,
+    keepAliveTime: Option[Duration] = None,
+    clientDuration: Option[Duration] = None
```

Comment:
I don't think we need that one. Apollo Server only has the keep-alive parameter.